includes/Utilities/Plugin_Request_Utility.php

```diff
@@ -134,4 +134,29 @@ public static function destroy_runner() {
 
 		static::$runner = null;
 	}
+
+	/**
+	 * Gets the directories to ignore using the filter.
+	 *
+	 * @since n.e.x.t
+	 */
+	public static function get_directories_to_ignore() {
+		// By default, ignore the '.git', 'vendor' and 'node_modules' directories.
+		$default_ignore_directories = array(
+			'.git',
+			'vendor',
+			'node_modules',
```

Comment:
This is a good initial list, but after seeing [your comment on the original issue](https://github.com/10up/plugin-check/issues/224#issuecomment-1658130784) after running this, I wonder if we should expand this to include other hidden files, or update the checks to ensure they're only running against the types of files we want to check, e.g. PHP?

Reply:
I personally don't think we have to add that because it's the result of the `File_Type_Check` check, and I ran checks in the development version of the plugin, and it makes sense to show those errors.

In the future, we can add possible values if we receive support tickets for that, or users can ignore them using the provided filter.

I would like to get @felixarntz feedback here.

Reply:
Sounds good to me. Thanks all.